test/end-to-end.js

```diff
@@ -1215,15 +1240,8 @@ function readBuiltinPhasesFromSanbox() {
   return app._requestHandlingPhases;
 }
 
-function bootSandboxWithOptions(options, done) {
-  var loopback = require(SANDBOX + '/node_modules/loopback');
-  var boot = require(SANDBOX + '/node_modules/loopback-boot');
-  var app = loopback({ localRegistry: true, loadBuiltinModels: true });
-  var bootOptions = extend({
-    appRootDir: SANDBOX + '/server',
-  }, options);
-
-  boot(app, bootOptions, function(err) {
-    done(err, app);
-  });
+function getSandBoxApp() {
```

Comment:
@superkhau proposed in an earlier comment to drop `sandbox` from the name. I am proposing to replace `get` with `load`. That would lead us to `loadApp` or perhaps `loadScaffoldedApp` to make it more clear what kind of application we are loading.